go-controller/pkg/ovn/default_network_controller.go

```diff
@@ -711,6 +713,7 @@ func (h *defaultNetworkControllerEventHandler) IsResourceScheduled(obj interface
 // Given an object to add and a boolean specifying if the function was executed from iterateRetryResources
 func (h *defaultNetworkControllerEventHandler) AddResource(obj interface{}, fromRetryLoop bool) error {
 	var err error
+	var aggregatedErrors []error
```

Comment:
same thing... why is it not under Node case?

Reply:
Moved it under NodeType case, Thanks!